{"name": "Try Signup With Invalid Username", "status": "passed", "steps": [{"name": "Регистрируемся", "status": "passed", "parameters": [{"name": "username", "value": "'nkNLvixxqfViEivseEMbxeofXVumJkpgBfqdGYaCpTdeQrqMxyE'"}, {"name": "password", "value": "'5fMkeMx_+)'"}], "start": 1751800823509, "stop": 1751800823782}], "start": 1751800823388, "stop": 1751800823782, "uuid": "af609529-6ba4-4b96-a858-f3ee14e016ae", "historyId": "5bf71bf0cb3efa99e9ce6fbe2d86867e", "testCaseId": "5bf71bf0cb3efa99e9ce6fbe2d86867e", "fullName": "tests.test_auth.TestsSignup#test_try_signup_with_invalid_username", "labels": [{"name": "epic", "value": "Форма регистрации"}, {"name": "feature", "value": "Регистрация"}, {"name": "story", "value": "Попытка регистрации под невалидным именем пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestsSignup"}, {"name": "host", "value": "WIN-V8TQK9SAURU"}, {"name": "thread", "value": "24696-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}